### PR TITLE
fix: Add close button to QR Code Reader

### PR DESCRIPTION
**Pull request by ChangoMan**
## Description

This is a way to fix https://github.com/BuidlGuidl/burnerwallet/issues/65

The issue is, when the Send Drawer is open, it manipulates other parts of the website so nothing else can be interacted with. Only the elements in the drawer can be interacted with.

I'm adding a close button to the QR code reader, with `pointer-events-auto` that will override this. _However_, when you close the QR code reader it will also close the Send modal now. I'm not sure how to prevent this, but this is a solution to the issue mentioned above.

### CODE REVIEW

File: packages/nextjs/components/burnerwallet/QrCodeReader.tsx [hunk 1]
```diff
@@ -54,6 +55,9 @@ const QrCodeReader = () => {
             </div>
           ) : (
             <div className="max-w-[90%] w-[300px] h-[300px] fixed top-0 left-0 right-0 bottom-0 m-auto z-[100]">
+              <button className="pointer-events-auto" onClick={() => setIsQrReaderOpen(false)}>
+                <XCircleIcon className="w-6 h-6" />
```
Comment (Pabl0cks) [inline packages/nextjs/components/burnerwallet/QrCodeReader.tsx:59]: Is not showing to me in light mode, should we force the font color to white?

```suggestion
                <XCircleIcon className="w-6 h-6 text-white" />
```

File: packages/nextjs/components/burnerwallet/QrCodeReader.tsx [hunk 2]
```diff
@@ -54,6 +55,9 @@ const QrCodeReader = () => {
             </div>
           ) : (
             <div className="max-w-[90%] w-[300px] h-[300px] fixed top-0 left-0 right-0 bottom-0 m-auto z-[100]">
+              <button className="pointer-events-auto" onClick={() => setIsQrReaderOpen(false)}>
```
Comment (Pabl0cks) [inline packages/nextjs/components/burnerwallet/QrCodeReader.tsx:58]: Nitpick: to me feels more natural the X to close at the right side of the camera square. Maybe just personal preference!

**Review by Pabl0cks — COMMENTED** [review]
The solution looks good to me @ChangoMan! Added a couple of comments 🙌

File: packages/nextjs/components/burnerwallet/QrCodeReader.tsx (hunk 1 above)
Comment (ChangoMan) [reply packages/nextjs/components/burnerwallet/QrCodeReader.tsx]: Good call, got this fixed and aligned the button to the right

**Review by Pabl0cks — APPROVED** [review]
Looking good!

**Pull request merged**